spec/controllers/reports_controller_spec.rb

```diff
@@ -92,9 +92,41 @@
     before :each do
       session[:user_id] = users(:bob_smith_user).id
     end
+
     it "should show the form" do
       get :new, :request_id => info_request.url_title
       expect(response).to render_template("new")
     end
+
+    it "accepts a comment id" do
+      get :new, :request_id => info_request.url_title, :comment_id => 1
+      expect(response).to render_template("new")
+    end
+
+    context "when passed a comment id" do
+      render_views
+
+      let(:comment) do
+        FactoryGirl.create(:comment, :info_request => info_request,
+                                     :body => "This is a vexations comment")
+      end
+
+      it "ignores the comment id if it does not belong to the request" do
+        new_request = FactoryGirl.create(:info_request)
+        new_comment = FactoryGirl.create(:comment, :info_request => new_request,
+                                                   :body => "Some comment")
```

Comment:
There are a few cases of this – no need to introduce extra context thats not relevant to the given spec.